x-pack/plugin/core/template-resources/src/main/resources/logs@settings-logsdb.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
why are these templates moved to logsdb module? I think these templates should stay in the stack module?

Reply:
The new `logs@settings-logsdb` template is required because we can have two cases:
1. "mode": "logsdb" when logsdb is enabled
2. no `index.mode` at all. In `LogsdbIndexModeSettingsProvider` the logic expect to have empty/null `index.mode` to inject the "logsdb" index mode.

The way it is the template now always assigns `index.mode` a value, "logsdb" or "standard". As a result injection would never work in `LogsdbIndexModeSettingsProvider`.

I believe if this setting does not work in `StackTemplateRegistry` is it better to remove it...despite requiring more work...especially because some tests might rely on it. If you agree I would do the removal in a separate PR to keep the scope of this one a bit more limited.

Reply:
Ok, so we said we temporarily add this template file so the `xpack.stack.template.logsdb.index.mode` template variable can be removed. Otherwise standard would always be set as index mode and there is no way to know whether this was set specifically or a default.

After this PR is merged we will remove the `cluster.logsdb.enable` setting logic and this templayte from the stack module. Which makes reviewing that change easier to review.